contracts/vortex/CarbonVortex.sol

```diff
@@ -1012,6 +1032,19 @@ contract CarbonVortex is ICarbonVortex, Upgradeable, ReentrancyGuardUpgradeable,
         return price;
     }
 
+    function _transferProceeds(Token token, uint256 amount) private {
+        // if transfer address is 0, proceeds stay in the vortex
+        if (_transferAddress == address(0)) {
```

Comment:
Why allowing the funds to stay in the Vortex is a good idea?

Reply:
It’s actually a requirement. The next phase would be to allow sending them to a bridging contract and then to Ethereum mainnet

Reply:
Roger that